client/cl_boss.lua

```diff
@@ -3,6 +3,8 @@ local PlayerJob = QBCore.Functions.GetPlayerData().job
 local shownBossMenu = false
 local DynamicMenuItems = {}
 
+
+
```

Comment:
I'm not sure that these spacing lines are needed as the utility functions are already somewhat spaced from the variable declarations.